src/config/progress_options.rs

```diff
@@ -42,7 +48,8 @@ impl ProgressOptions {
     ///
     /// `Duration::ZERO` if no progress is enabled
```

Comment:
```suggestion
    /// `Duration::ZERO` if no progress is enabled
```

Does that need an update as well?

Reply:
is changed.